### PR TITLE
Bump to 0.4.1 (again)

### PR DESCRIPTION
**Pull request by fehiepsi**
@neerajprad I missed this place. Without this, upload to PyPI raise some errors. :(

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
             'pyro-api>=0.1.1'
         ],
         'dev': [
-            'funsor @ git+https://github.com/pyro-ppl/funsor.git@6575ac2c3f7ac25a6a1e5f1107b1b7c072edd992',
+            'funsor',
```
Comment (neerajprad) [inline setup.py:50]: So the latest release of funsor has this commit?

Comment (fehiepsi) [reply setup.py:50]: oh, good question! I need to verify this...

Comment (fehiepsi) [reply setup.py:50]: yes, it has.

**Review by neerajprad — APPROVED** [review]

**Comment by fehiepsi** [comment]
Thanks, @neerajprad!

**Pull request merged**